assignment8.txt

```diff
@@ -0,0 +1,36 @@
+/*Rebecca Hale Assignment 8 Part 2*/
+
+1. fprintf() and printf() are both functions used in C to print out given
+input.  However, they are slightly different.  fprintf() prints input
+through a specific stream (FILE) while printf() sends formatted output to
+the stdout stream.  Both are of type int, so they technically return an int
+when used.  Both printing methods can utilize format specifiers to input
+specific data types but generally they are both given string input when no
+specifiers are used.
+
+2. Opening a file in "rw" mode indicates that the computer will read from
```

Comment:
mostly right! The only thing that's different is that "a" only adds to the _end_ of a file.